resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfigurationNotFound.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *
+ *  Copyright 2019 Mahmoud Romeh
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *
+ */
+package io.github.resilience4j.circuitbreaker;
+
+/**
+ * configuration not found exception
+ */
+public class CircuitBreakerConfigurationNotFound extends RuntimeException {
```

Comment:
Can we reanme it to `ConfigurationNotFoundException` and move it into resilience4j-core.
We can reuse it for other Registries as well.